CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+Include any required SharePoint changes that will be needed to deploy you PR
+Update the version number in package.json when submitting your PR
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+
+## [Unreleased]
+- (Keep your changes here until you have a release version)
+
+## [0.1.0] - 2022-05-06
+### Added
+- This CHANGELOG file.
```

Comment:
@RobertPorterfield -- Should you have some notes here?

Reply:
Yes I absolutely should! 😆 
I'll add a summary